src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        int ballsLength = balls.length;
+
```

Comment:
remove empty line

Reply:
I did between constructions indent because it looks better and more readable on my opinion

Reply:
Remove it anyway)